go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/minio/operator
 
-go 1.21.8
+go 1.21.11
```

Comment:
Are there known issues with <1.21.11 that require an upgrade?

Reply:
No idea.